packages/components/src/components/drawer/drawer.scss

```
@@ -89,6 +90,7 @@ $spacings: (
 
 	.db-drawer-content {
 		// We need this variable to overwrite it inside the header
+		/* stylelint-disable-next-line db-ui/use-spacings */
```

```suggestion
		/* TODO: We would need to correct this reference in the future */
		/* stylelint-disable-next-line db-ui/use-spacings */
```

this is a potential TODO, isn't it ?

---

https://github.com/db-ui/mono/pull/3727#discussion_r1939315921